frameworks/Ruby/roda-sequel/README.md

```diff
@@ -12,10 +12,10 @@ comparing a variety of web platforms.
 
 The tests will be run with:
 
-* [Ruby 3.3](http://www.ruby-lang.org)
+* [Ruby 3.4](http://www.ruby-lang.org)
 * [Puma 6](http://puma.io)
-* [Passenger 6](https://www.phusionpassenger.com)
-* [Unicorn 5](https://bogomips.org/unicorn/)
+* [Unicorn 6](https://bogomips.org/unicorn/)
```

Comment:
Did you intend for Unicorn to be bumped to 6?

Reply:
Yes, it has been using 6 for a while, but the README was never updated.
https://github.com/TechEmpower/FrameworkBenchmarks/blob/68a50181ca98747a2e507eb2b935f6d35987bd6f/frameworks/Ruby/roda-sequel/Gemfile#L11